dcat/index.html

```diff
@@ -2703,8 +2703,9 @@ <h4>Property: byte size</h4>
                 <tbody>
                 <tr><td class="prop">Definition:</td><td>The size of a distribution in bytes.</td></tr>
                 <tr><td class="prop">Domain:</td><td><a href="#Class:Distribution"><code>dcat:Distribution</code></a></td></tr>
-                <tr><td class="prop">Range:</td><td><a href="http://www.w3.org/2000/01/rdf-schema#Literal"><code>rdfs:Literal</code></a> typed as <a data-cite="XMLSCHEMA11-2#decimal"><code>xsd:decimal</code></a>.</td></tr>
-                <tr><td class="prop">Usage note:</td><td>The size in bytes can be approximated (as a decimal) when the precise size is not known.</td></tr>
+                <tr><td class="prop">Range:</td><td><a href="http://www.w3.org/2000/01/rdf-schema#Literal"><code>rdfs:Literal</code></a> typically typed as <a data-cite="XMLSCHEMA11-2#integer"><code>xsd:integer</code></a>.</td></tr>
```

Comment:
If we are going to change it, then probably we should go for `xsd:positiveInteger` or `xsd:nonNegativeInteger`.